runtime/libpgmath/lib/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 
 # This value will be the same as LIBPGMATH_SYSTEM_PROCESSOR
 set(PROCESSOR ${LIBPGMATH_SYSTEM_PROCESSOR})
-if(${LIBPGMATH_WITH_GENERIC} OR (NOT ${LIBPGMATH_SYSTEM_PROCESSOR} MATCHES "x86_64|aarch64"))
+if(${LIBPGMATH_WITH_GENERIC} OR (NOT ${LIBPGMATH_SYSTEM_PROCESSOR} MATCHES "x86_64|aarch64|ARM64"))
```

Comment:
Come to think of it, why is  the ARM64 introduced at all? The CMake script should check for `aarch64 AND WIN32` or similar.